src/mouse_events.cpp

```diff
@@ -79,6 +79,7 @@ mouse_handler::mouse_handler(game_display* gui, play_controller & pc) :
 	over_route_(false),
 	reachmap_invalid_(false),
 	show_partial_move_(false)
+	, preventing_units_highlight_(false)
```

Comment:
While I agree that putting the comma at the beginning is preferred, I think it's better to match the style of the surrounding code. (But maybe wait for an opinion on @Vultraz here.)

Reply:
I am neutral to _comma before_ vs _comma after_ in most cases, except SQL scripts, and I think it's good to match the style of the surrounding code. I think it's good deferring `git annotate` changes on lines until completely necessary.

Here is just that the entire commit was originally conceived as changing as few lines as possible, to make a peek on the maintainer about their preferences when it comes to review and when it comes to merge PR.

Will comply request unless @Vultraz comments against.

Reply:
Yeah, the idea of changing as few lines as possible is a good one; I try to do the same in my commits and would have likely even done the same as you did in that section where a block needs re-indenting.

Reply:
Too bad, GitHub is buggy here, it validates a comment on diff whose context lines changed.

D: